app/src/main/res/values/themes.xml

```diff
@@ -1,8 +1,8 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.H3LP" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+    <style name="Theme.H3LP" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_500</item>
+        <item name="colorPrimary">@color/light_grey</item>
```

Comment:
I think theres no need to change that for now since we have not really decide the grafical chart.

I would anyway propose teal since it is the one in the background image of most of the page